src/Explorer/QueryCopilot/QueryCopilotPromptbar.tsx

```diff
@@ -539,84 +539,90 @@ export const QueryCopilotPromptbar: React.FC<QueryCopilotPromptProps> = ({
       </Stack>
 
       {showFeedbackBar && (
-        <Stack style={{ backgroundColor: "#FFF8F0", padding: "2px 8px" }} horizontal verticalAlign="center">
-          <Text style={{ fontWeight: 600, fontSize: 12 }}>Provide feedback on the query generated</Text>
-          {showCallout && !hideFeedbackModalForLikedQueries && (
-            <Callout
-              role="status"
-              style={{ padding: 8 }}
-              target="#likeBtn"
-              onDismiss={() => {
-                setShowCallout(false);
-                SubmitFeedback({
-                  params: {
-                    generatedQuery: generatedQuery,
-                    likeQuery: likeQuery,
-                    description: "",
-                    userPrompt: userPrompt,
-                  },
-                  explorer,
-                  databaseId,
-                  containerId,
-                  mode: isSampleCopilotActive ? "Sample" : "User",
-                });
-              }}
-              directionalHint={DirectionalHint.topCenter}
-            >
-              <Text>
-                Thank you. Need to give{" "}
-                <Link
-                  onClick={() => {
+        <Stack
+          style={{ backgroundColor: "#FFF8F0", padding: "2px 8px", minHeight: 32 }}
+          horizontal
+          verticalAlign="center"
+        >
+          {userContext.feedbackPolicies?.policyAllowFeedback && (
```

Comment:
Does this mean that if we don't have feedbackPolicies in the userContext then we won't display feedback ?

Reply:
Yeah. I think this will only apply if someone decides to use copilot in standalone DE.

Reply:
Or what happens if the  customer doesn't have feedback policies defined for their tenant. Will it be null in that case ?

Reply:
Ah, the feedback policy API from the portal SDK will return object with all True values if they dont have anything configured. So that shouldn't be a case.

Reply:
Got it - cool. But we still need to support the standalone DE scenario ?